docs/hopsworksai/azure/upgrade.md

```diff
@@ -139,8 +139,66 @@ Navigate back to the your Resource group home page at [Azure portal](https://por
 !!! warning
     [When you assign roles or remove role assignments, it can take up to 30 minutes for changes to take effect.](https://docs.microsoft.com/en-us/azure/role-based-access-control/troubleshooting#role-assignment-changes-are-not-being-detected)
 
+## Step 3: Add disk read permissions to your role connected to Hopsworks.ai
 
-## Step 3: Run the upgrade process
+We require extra permission ("Microsoft.Compute/disks/read") to be added to the role you used to connect to Hopsworks.ai, the one that you have created in [Getting started Step 1.2](../getting_started/#step-12-creating-a-custom-role-for-hopsworksai). 
+If you don't remember the name of the role that you have created in [Getting started Step 1.2](../getting_started/#step-12-creating-a-custom-role-for-hopsworksai), you can navigate to your Resource group, (1) click on *Access Control*, (2) navigate to the *Check Access* tab, (3) search for *hopsworks.ai*, (4) click on the one with the blue icon, (5) now you have the name of your custom role used to connect to hopsworks.ai. 
```

Comment:
The users will not have the hopsworks.ai with the black icon, so I think you can replace `(4) click on the one with the blue icon` by `click on it` and hide the unwanted one in the screenshot.

Reply:
fixed